src/components/PokeCard.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+
+export default class PokeCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { surprise: false };
+  }
+
+  render() {
+    return (
+      <div>
+        <img
+          src={this.props.sprites.front_default}
```

Comment:
you could use destructuring at the beginning of render() as in:
```js
const { name, sprites, video } = this.props;
```
Using them later at the rest of the function makes your code easier to read.

Reply:
that would be way better, thanks! :D